spec/models/user_model_spec.rb

```diff
@@ -47,6 +47,7 @@
 
   describe "ClassMethods" do
     before :all do
+      DatabaseCleaner.start
```

Comment:
Hmmm....

What I have in mind is having a single `DatabaseCleaner.start` and a single `DatabaseCleaner.end` hooked directly into the spec config... E.g as follow:

```
...
  config.before(:each) do
    DatabaseCleaner.start
  end

  config.after(:each) do
    DatabaseCleaner.clean
  end
...
```

Then we won't need to call it individually everywhere we need it again.... if just works for all specs